### ISSUE TITLE
Trimester Three! Week 25 Updates and Improvements

### ISSUE THREAD

**Issue by TheGerbil21** [TheGerbil21/firstfastpages#27]
y

**Comment by jagermi3ster** [comment]
2.7+ 

Jeffrey showed a great understanding of the unit. He completed both 2.2 and 2.3 blogs with hacks. He didn't have the 2.2 and 2.3 college board practice problems, nor did he implement his hacks into his final issues. Show good effort.

**Comment by TheGerbil21** [comment]
> 2.7+
> 
> Jeffrey showed a great understanding of the unit. He completed both 2.2 and 2.3 blogs with hacks. He didn't have the 2.2 and 2.3 college board practice problems, nor did he implement his hacks into his final project. Shown good effort.

I agree with Jagger's grading, as I did put a good amount of time and effort into my hacks for 2.2 and 2.3. However, I chose not to implement these changes into my final project until I have a better understanding of the content. I also still need to do the CB practice problems for both notebooks. Overall, I don't reach the 2.7+ territory but showed good Scrum Master skills and a thorough Team Review Ticket for a higher grade.
